code/modules/mob/living/simple_animal/abnormality/aleph/mountain.dm

```diff
@@ -78,6 +79,47 @@
 		return FALSE
 	return ..()
 
+/mob/living/simple_animal/hostile/abnormality/mountain/Moved()
+	. = ..()
+	var/turf/floor = get_turf(src)
+	for(var/obj/thing in floor)
+		if(istype(thing, /obj/effect/decal/cleanable/blood))
+			var/obj/effect/decal/cleanable/blood/blood_decal = thing
+			adjustBruteLoss(-clamp((maxHealth*0.005)*(blood_decal.bloodiness*0.01), 0, maxHealth)) // Heal up to 0.5% heath from bloody ground.
+			QDEL_NULL(blood_decal)
+			belly += 0.01
+			StageChange(TRUE)
+			continue
+		if(istype(thing, /obj/item/organ))
+			var/obj/item/organ/floor_organ = thing
+			adjustBruteLoss(-clamp((maxHealth*0.01), 0, maxHealth)) // Heal 1% health from Organs
+			QDEL_NULL(floor_organ)
+			belly += 0.05
+			StageChange(TRUE)
+			continue
+		if(istype(thing, /obj/item/bodypart))
+			var/obj/item/bodypart/floor_part = thing
+			adjustBruteLoss(-clamp((maxHealth*0.01), 0, maxHealth)) // Heal 1% health from Body Parts
+			QDEL_NULL(floor_part)
+			belly += 0.05 // Progresses toward getting bigger
+			StageChange(TRUE)
+			continue
+	new /obj/effect/decal/cleanable/old_flesh(floor)
+
+/mob/living/simple_animal/hostile/abnormality/mountain/apply_damage(damage, damagetype, def_zone, blocked, forced, spread_damage, wound_bonus, bare_wound_bonus, sharpness, white_healable)
+	. = ..()
+	var/messyArea = list()
+	var/validArea = view(2, src)
+	for(var/I = 1 to rand(1, 3))
+		var/turf/space = pick_n_take(validArea)
+		if (!(space in messyArea) && !space.density)
+			messyArea += space
+		else
+			I -= 1
+	playsound(src, 'sound/effects/blobattack.ogg', 75)
+	for(var/turf/T in messyArea)
+		new /obj/effect/decal/cleanable/old_flesh(T)
```

Comment:
This will get REALLY annoying, and we already saw it in action and yes, it was REALLY annoying.
Either check for amount of damage or make it not always proc.

Reply:
Made it a 25% chance